pkg/utils.py

```diff
@@ -20,6 +20,8 @@
 import pyshorteners
 from pkg.constants import NANO_SECONDS_PATTERN
 
+uuid_field="uuid"
```

Comment:
If these variables need to be preset, can we convert them into instance variables?